### PR TITLE
Improved logging for Flash

### PR DESCRIPTION
**Pull request by omerfirmak**
courtesy of drey

### CODE REVIEW

File: source/agora/test/QuorumPreimage.d [hunk 1]
```diff
@@ -49,7 +49,7 @@ unittest
     scope(exit) network.shutdown();
     // Disabled, as most of the time they just hide the more useful
     // quorum prints which is also a `scope (failure)` (see below)
-    // scope(failure) network.printLogs();
+    scope(failure) network.printLogs();
```
Comment (Geod24) [inline source/agora/test/QuorumPreimage.d]: Is the comment wrong nowadays ?

File: source/agora/test/Flash.d [hunk 2]
```diff
@@ -277,6 +279,29 @@ public class TestFlashNode : ThinFlashNode, TestFlashAPI
...
+                        writefln("cannot find direction %s", dir);
+                    }
+                }
+                else
+                {
+                    log.info("Can't find chan ID {}. map: {}",
+                        chan_id, cast(void*)&this.channel_updates);
+                }
+            }
+
```
Comment (Geod24) [inline source/agora/test/Flash.d]: This looks like debugging leftover ?

Comment (omerfirmak) [reply source/agora/test/Flash.d]: Oops, I thought they were all logs. Will take this one out.

File: source/agora/test/QuorumPreimage.d (hunk 1 above)
Comment (omerfirmak) [reply source/agora/test/QuorumPreimage.d]: Well, I am not sure what his intention was. but that comment is obviously no longer useful.

**Review by Geod24 — APPROVED** [review]

**Pull request merged**